lighthouse-core/test/report/html/renderer/report-renderer-test.js

```diff
@@ -255,7 +255,8 @@ describe('ReportRenderer', () => {
 
     let notApplicableCount = 0;
     Object.values(clonedSampleResult.audits).forEach(audit => {
-      if (audit.scoreDisplayMode === 'notApplicable') {
+      // The performance-budget audit is omitted from the DOM if it is not applicable
```

Comment:
I missed that we were building the actual expected count since that was down on 271 and I just looked at 265, WDYT

```suggestion
      // The performance-budget audit is omitted from the DOM when it is not applicable, so omit from our expected count too.
```